### PR TITLE
[#445] Add re:dash embeds as real time database statistics

### PR DESCRIPTION
**Pull request by nightsh**
Fixes opentrials/opentrials#445

### CODE REVIEW

**Comment by vitorbaptista** [comment]
@nightsh Looks like you forgot to commit some files

**Comment by nightsh** [comment]
Oh, sorry, you're right! 😅 
Pushed them all now.

File: handlers/stats.js [hunk 1]
```diff
@@ -1,19 +1,12 @@
...
 function getStats(request, reply) {
-  stats.get().then((_stats) => {
-    reply.view('stats', {
-      title: 'Statistics',
-      stats: _stats,
-    });
-  }).catch((err) => {
-    reply(Boom.badGateway('Error accessing OpenTrials API.', err));
+  reply.view('stats', {
+    title: 'Stats',
```
Comment (vitorbaptista) [inline handlers/stats.js]: Rename title to Statistics

File: views/stats.html [hunk 2]
```diff
@@ -7,56 +7,17 @@
...
-        <dd>{{ sourceLatestUpdatedDate.latest_updated_date | formatDate }}</dd>
-      </li>
-      {% endfor %}
-    </ul>
-  </div>
+  <!-- This is the big counter, disabled for now since it might be too big for this page
+  <iframe id="stats-counter"
+          src="http://query.opentrials.net/embed/query/1/visualization/11?api_key=c6a1ba37f32b54809383498d0bd4cb380677ad9f"
+  ></iframe>
+  -->
```
Comment (vitorbaptista) [inline views/stats.html]: Remove the commented code.

File: assets/styles/_stats.scss [hunk 3]
```diff
@@ -0,0 +1,14 @@
+iframe {
+  width: 100%;
+  border: none;
+  overflow: hidden;
+  &#stats-counter {
+    height: 40em;
+  }
```
Comment (vitorbaptista) [inline assets/styles/_stats.scss]: There's no counter anymore, please remove it.

**Pull request merged**